static/js/tippyjs.js

```diff
@@ -174,6 +175,22 @@ export function initialize() {
         e.currentTarget._tippy.destroy();
     });
 
+    delegate("body", {
+        target: "#message_type_icon",
+        appendTo: () => document.body,
+        onShow(instance) {
+            const $elem = $(instance.reference)[0];
+            instance.setContent(
+                $elem.classList.contains("is_pm")
+                    ? compose_actions.switch_to_stream_message
+                    : compose_actions.switch_to_pm_message,
```

Comment:
I'd prefer an if/else block here -- it's a better default pattern because code coverage tools can pick up whether one's tests cover both cases.